core/xchain/create_multi.sh

```diff
@@ -59,6 +59,8 @@ BASE_DIR="./run_files/${RUN_ENV}"
 
 mkdir -p "${BASE_DIR}"
 
+source ../../contracts/.env.localhost
```

Comment:
@brianathere I made the following changes to get xchain to properly register itself on startup, to mimic how stream node registers when the service is being launched (using the same wallet as the operator address), but was unsuccessful.

Reply:
Right now xchain isn't launching, and it's failing on the registration step. This is why the skd tests for space admission that require xchain in the loop are failing.